learning_journal/views/default.py

```diff
@@ -3,25 +3,56 @@
 
 from sqlalchemy.exc import DBAPIError
 
-from ..models import MyModel
+from ..models import Entry
 
 
-@view_config(route_name='home', renderer='../templates/mytemplate.jinja2')
+@view_config(route_name='list', renderer='../templates/list.jinja2')
 def my_view(request):
     try:
-        query = request.dbsession.query(MyModel)
-        one = query.filter(MyModel.name == 'one').first()
+        query = request.dbsession.query(Entry)
+        entries = query.all()
     except DBAPIError:
         return Response(db_err_msg, content_type='text/plain', status=500)
-    return {'one': one, 'project': 'learning_journal'}
+    return {'entries': entries, 'project': 'learning_journal'}
+
+
+@view_config(route_name='detail', renderer='../templates/detail.jinja2')
+def detail_view(request):
+    try:
+        query = request.dbsession.query(Entry)
+        the_id = int(request.matchdict["id"])
+        entry = query.all()[the_id]
+    except DBAPIError:
+        return Response(db_err_msg, content_type='text/plain', status=500)
+    return {'entry': entry, 'project': 'learning_journal'}
 
 
 @view_config(route_name='create', renderer='../templates/create.jinja2')
 def create_view(request):
     if request.method == "POST":
-        # get the form stuff
-        pass
-    return {}
+        new_title = request.POST["title"]
+        new_post = request.POST["post"]
+        new_model = Entry(title=new_title, body=new_post)
+        request.dbsession.add(new_model)
+        return {"data": {"title": "post"}}
+    return {"data": {"title": "post"}}
```

Comment:
Why does your create return data?  Create should be a blank form that takes data and creates an entry.  Update/edit and detail will want data returned.